pkgs/tools/misc/gtkclipblock/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, cmake
+, ninja
+, pkg-config
+, gtk2
+, gtk3
+, gtk4
+}:
+let
+  distorm-src = fetchFromGitHub {
+    owner = "gdabah";
+    repo = "distorm";
+    rev = "3.5.2b";
+    hash = "sha256-2ftEV3TMS3HT7f96k+Pwt3Mm31fVEXcHpcbbz05jycU=";
+  };
+  source = fetchFromGitHub {
+    owner = "notpeelz";
+    repo = "gtkclipblock";
+    rev = "v0.2.2";
+    hash = "sha256-ok/D7M0KekN8zf8AzhcOLtedbYVRHHv3m9zEHsJfcPM=";
+    fetchSubmodules = true;
+  };
+in
+stdenv.mkDerivation {
+  pname = "gtkclipblock";
+  version = "0.2.2";
+  src = source;
+  nativeBuildInputs = [ meson cmake ninja pkg-config ];
```

Comment:
Why the are you adding cmake AND meson?

Reply:
Because `gtkclipblock` submodules and builds `funchook-helper`, which submodules and builds `funchook`, which is a CMake project.

Reply:
This can be confusing, since both Meson and Cmake modify the build and install phases, and I don't know which one will be executed.

Reply:
Is there any way to make it less confusing/pass CMake to meson in a way that won't modify the build/install phases? Or at least get the commands that run each build/install to see what happens?

I am also a bit confused as to how that ambiguity affects us practically speaking, because I definitely know the package builds/installs correctly (using meson) and the artifact produced works as advertised...

Reply:
I will ask about it on Discourse.

Reply:
Usually it is done via `dontUseCmakeConfigure`.

Reply:
Thanks! I've set it.